src/components/PhoneInput/__tests__/phoneinput.spec.js

```diff
@@ -32,4 +32,20 @@ describe('<PhoneInput />', () => {
         const wrapper = mount(<PhoneInput countries={countries} />);
         expect(wrapper.find(StyledTrigger).prop('onClick')).toBe(undefined);
     });
+
+    it('should render the dropdown option when an array containing empty string is passed', () => {
+        const countries = [''];
+        const onClickMockFn = jest.fn();
+        const wrapper = mount(<PhoneInput countries={countries} onClick={onClickMockFn} />);
+        wrapper.find(StyledTrigger).simulate('click');
+        expect(onClickMockFn).toBeDefined();
+    });
+
+    it('should render the dropdown option when an array containing invalid country code is passed', () => {
+        const countries = [!/^\w{1,2}$/];
+        const onClickMockFn = jest.fn();
+        const wrapper = mount(<PhoneInput countries={countries} onClick={onClickMockFn} />);
+        wrapper.find(StyledTrigger).simulate('click');
+        expect(onClickMockFn).toBeDefined();
+    });
```

Comment:
here we got the changes made in other PR, this can happen when creating a new branch from others with these changes, normally we should make branch always form master in order to avoid this